programs/launcher/main.cpp

```diff
@@ -716,8 +716,7 @@ launcher_def::deploy_config_file (tn_node_def &node) {
            << " errno " << ec.value() << " " << strerror(ec.value()) << endl;
```

Comment:
Above this, at line 468, can we add:

          } catch (fc::exception& fce) {
            cerr << "unable to launch " << inst.name << "fc::exception=" << fce.to_detail_string() << endl;
          } catch (std::exception& stde) {
             cerr << "unable to launch " << inst.name << "std::exception=" << stde.what() << endl;

Reply:
It helped me out a lot when it was failing because the test directory wasn't cleaned up previously for some reason.